src/skeletons.css.ts

```diff
@@ -0,0 +1,48 @@
+import {sprinkles} from './sprinkles.css';
+import {style, keyframes} from '@vanilla-extract/css';
+
+const transition = '1.5s linear';
+
+const pulseKeyframes = keyframes({
+    '0%': {
+        opacity: 1,
+    },
+
+    '50%': {
+        opacity: 0.5,
+    },
+
+    '100%': {
+        opacity: 1,
+    },
+});
+
+export const animation = style([
+    sprinkles({
+        width: '100%',
+        height: '100%',
```

Comment:
maybe these two could be removed? we found that issue that reported smorchon about skeletons alignment

Reply:
it requires more changes, better review that in a different task. I don't know if there are other implications
 https://jira.tid.es/browse/WEB-709